recipe/cross_compile_support.sh

```diff
@@ -29,12 +29,16 @@ if [[ "${HOST_PLATFORM}" != "${BUILD_PLATFORM}" ]]; then
         echo "- ${BUILD_PLATFORM}"   >> ${CI_SUPPORT}/${CONFIG}.yaml
     fi
     if [[ "${BUILD_PLATFORM}" == "linux-64" && "${HOST_PLATFORM}" == linux-* ]]; then
-        mamba create -n sysroot_${HOST_PLATFORM} --yes --quiet sysroot_${HOST_PLATFORM}=${GLIBC_VERSION}
+        micromamba create -n sysroot_${HOST_PLATFORM} --root-prefix ~/.conda --yes --quiet \
+            --override-channels --channel conda-forge --strict-channel-priority \
+            sysroot_${HOST_PLATFORM}=${GLIBC_VERSION}
         HOST_PLATFORM_ARCH=${HOST_PLATFORM:6}
         if [[ -f ${RECIPE_ROOT}/yum_requirements.txt ]]; then
             cat ${RECIPE_ROOT}/yum_requirements.txt | while read pkg; do
                 if [[ "${pkg}" != "#"* && "${pkg}" != "" ]]; then
-                    mamba install "${pkg}-conda-${HOST_PLATFORM_ARCH}" -n sysroot_${HOST_PLATFORM} --yes --quiet || true
+                    micromamba install -n sysroot_${HOST_PLATFORM} --yes --quiet \
```

Comment:
I think you need `--root-prefix` here too.

Reply:
I didn't add it the second time because the warning said:
```
To permanently modify the root prefix location, either:
      - set the 'MAMBA_ROOT_PREFIX' environment variable
      - use the '-r,--root-prefix' CLI option
      - [...]
```
The "permanently" sounded like once is enough